test/release/examples/primers/arrays.chpl

```diff
@@ -99,6 +99,27 @@ forall (i,j) in {1..n, 1..n} do
 
 writeln("Initially, B is:\n", B, "\n");
 
+//
+// Multidimensional arrays can also be created with array literals. They have
+// the same default domain and type semantics as 1D array literals, but with a
+// slightly different syntax. Multidimensional array literals are specified by
+// using semicolons to separate the rows of the array. For example, the
```

Comment:
It might make sense to change "rows" to "dimensions" here, as "rows" feels very 2D specific and since your example is 3D, they're also used to separate the "planes"/"levels" if you will.

Reply:
Whoops, too slow on the draw…

Reply:
It should be updated now